tests/test_aad_helpers.py

```diff
@@ -4,8 +4,8 @@
 from authlib.jose.rfc7519.claims import JWTClaims
 from helpers import gen_payload
 
-from aad import AadUser, AuthError, AzureAdSettings
-from aad.aad_helpers import _decode_token, _get_user_from_claims, _validate_claims
+from src.aad_fastapi import AadUser, AuthError, AzureAdSettings
+from src.aad_fastapi.aad_helpers import _decode_token, _get_user_from_claims, _validate_claims
```

Comment:
managed to run in w/o src. just aad_fastapi.aad_helpers

Reply:
fixed for all tests files (had to set up source root in IDE)